dotnet/src/IntegrationTests/Connectors/Memory/AzureCosmosDBNoSQL/AzureCosmosDBbNoSQLConnectionStringSetConditionAttribute.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading.Tasks;
+using SemanticKernel.IntegrationTests.Connectors.Memory.Xunit;
+
+namespace SemanticKernel.IntegrationTests.Connectors.Memory.AzureCosmosDBNoSQL;
+
+/// <summary>
+/// Attribute to use to skip tests if the connection string for CosmosDB NoSQL is not set.
+/// </summary>
+[AttributeUsage(AttributeTargets.Method | AttributeTargets.Class)]
+public sealed class AzureCosmosDBbNoSQLConnectionStringSetConditionAttribute : Attribute, ITestCondition
```

Comment:
Typo in file name and class name:
```suggestion
public sealed class AzureCosmosDBNoSQLConnectionStringSetConditionAttribute : Attribute, ITestCondition
```

Reply:
Thanks, fixed.